merchants/models.py

```diff
@@ -47,3 +51,242 @@ class PaymentProduct(models.Model):
 
     def __str__(self) -> str:
         return str(self.id)
+
+
+class PaymentType(models.TextChoices):
+    ON_SITE = "ON_SITE", _("On site")
+    ONLINE = "ONLINE", _("Online")
+    INVOICE = "INVOICE", _("Invoice")
+
+
+class PaymentStatus(models.TextChoices):
+    DRAFT = "DRAFT", _("Draft")
+    EXPIRED = "EXPIRED", _("Expired")
+    CANCELLED = "CANCELLED", _("Cancelled")
+    PAID = "PAID", _("Paid")
+    PAID_MANUALLY = "PAID_MANUALLY", _("Paid manually")
+    REFUNDED = "REFUNDED", _("Refunded")
+
+
+class Language(models.TextChoices):
+    FI = "fi", _("Finnish")
+    SV = "sv", _("Swedish")
+    EN = "en", _("English")
+
+
+class PaymentOrder(models.Model):
+    order_id = models.UUIDField(
+        verbose_name=_("Order ID"),
+        help_text=_("Talpa eCommerce order ID"),
+        blank=True,
+        null=True,
+    )
+    payment_id = models.CharField(
+        verbose_name=_("Payment ID"),
+        help_text=_("Talpa eCommerce payment ID"),
+        blank=True,
+        null=True,
+        max_length=128,
+    )
+    payment_type = models.CharField(
+        verbose_name=_("Payment type"),
+        blank=False,
+        null=False,
+        max_length=128,
+        choices=PaymentType.choices,
+    )
+    status = models.CharField(
+        verbose_name=_("Payment status"),
+        blank=False,
+        null=False,
+        max_length=128,
+        choices=PaymentStatus.choices,
+    )
+    price_net = models.DecimalField(
+        verbose_name=_("Net amount"),
+        max_digits=10,
+        decimal_places=2,
+    )
+
+    price_vat = models.DecimalField(
+        verbose_name=_("VAT amount"),
+        max_digits=10,
+        decimal_places=2,
+    )
+
+    price_total = models.DecimalField(
+        verbose_name=_("Total amount"),
+        max_digits=10,
+        decimal_places=2,
+    )
+
+    created_at = models.DateTimeField(
+        verbose_name=_("Created at"), null=False, default=timezone.now
```

Comment:
We could remove default= and add `auto_now_add=True` It is the way.
Then if you want to have some `updated_at` it would need `auto_now=True`

Reply:
Changed 👍